in_toto/keylib.go

```diff
@@ -152,3 +154,63 @@ func VerifySignature(key Key, sig Signature, data []byte) error {
 
 	return nil
 }
+
+/* Parse ed25519 from private json uses the passed json string to populate and
+* ed25519 key. These ed25519 keys have the format as generated using
+* in-toto-keygen:
+*  {
+    "keytype: "ed25519",
+    "scheme": "ed25519",
+    "keyid": ...
+    "keyid_hash_algorithms": [...]
+    "keyval": {
+        "public": "..." # 32 bytes
+        "private": "..." # 32 bytes
+    }
+   }
+*/
+func Parseed25519FromPrivateJSON(JSONString string) (Key, error) {
```

Comment:
It took me a few seconds to parse `Parseed...` (also because we deal with a `seed` below). Can we change this to `ParseEd...`?